packages/react-liveness/src/components/FaceLivenessDetector/shared/Toast.tsx

```diff
@@ -1,28 +1,46 @@
 import * as React from 'react';
-import { Flex, View, ViewProps } from '@aws-amplify/ui-react';
+import { Flex, View, ViewProps, useTheme } from '@aws-amplify/ui-react';
 
 import { LivenessClassNames } from '../types/classNames';
 
 interface ToastProps extends ViewProps {
   variation?: 'default' | 'primary' | 'error';
   size?: 'medium' | 'large';
   heading?: string;
+  isInitial?: boolean;
 }
 
+const toastStyle = {
+  borderRadius: 'unset',
+  fontSize: 'xxl',
+  padding: '0 var(--amplify-space-xs)',
+};
+
+const maxWidthStyle = { base: '100%' };
+
 export const Toast: React.FC<ToastProps> = ({
   variation = 'default',
   size = 'medium',
   children,
+  isInitial = false,
   ...rest
 }) => {
+  const { tokens } = useTheme();
   return (
     <View
       className={`${LivenessClassNames.Toast} ${LivenessClassNames.Toast}--${variation} ${LivenessClassNames.Toast}--${size}`}
-      maxWidth={{ base: '100%', small: '70%' }}
+      maxWidth={maxWidthStyle}
```

Comment:
`maxWidthStyle` set above is `{ base: '100%' }`, which is equivalent of saying `maxWidth="100%"` but doesn't have any extra style computations based on breakpoint.